docs/source/linchpincli_linchpin_init.rst

```diff
@@ -0,0 +1,20 @@
+Init
+====
+
+command : linchpin init
+
+This command initialises the directory structure to facilitate linchpin rise and linchpin drop commands.
+
+Options:
+  --path PATH  path for initialisation
+  --help       Help
+
+=========
+Examples:
+=========
+
++------------------------+---------------------------------------------------------+
+| Usage                  | Action                                                  |
++========================+=========================================================+
+| linchpin init          |  initialises the directory structure for rise & drop    |
```

Comment:
This optionally creates a PinFile, prompting if the PinFile already exists. The docs should reflect this detail.